change-beta/@azure-communication-react-a39e6ca4-c099-4621-8848-9bde8c53eb72.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
```

Comment:
changetype is prerelease while the feature is in beta

Reply:
I have pushed in another changelog that has the type as `prerelease` stated. But it did not overwrite this one.

Reply:
you will have to delete this file or edit it. sadly the changelog command cannot overwrite these files